pkg/utils/transform/transform.go

```diff
@@ -65,11 +65,48 @@ func InjectNamespaceConditional(preserveNamespace, targetNamespace string) mf.Tr
 		if ok && val == "true" {
 			return nil
 		}
-
 		return tf(u)
 	}
 }
 
+func InjectNamespaceRoleBindingSubjects(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		kind := strings.ToLower(u.GetKind())
+		if kind != "rolebinding" {
+			return nil
+		}
+		subjects, found, err := unstructured.NestedFieldNoCopy(u.Object, "subjects")
+		if !found || err != nil {
+			return err
+		}
+		for _, subject := range subjects.([]interface{}) {
+			m := subject.(map[string]interface{})
+			if _, ok := m["namespace"]; ok {
+				m["namespace"] = targetNamespace
+			}
+		}
+		return nil
+	}
+}
+
+func InjectNamespaceCRDWebhookClientConfig(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		kind := strings.ToLower(u.GetKind())
+		if kind != "customresourcedefinition" {
+			return nil
+		}
+		service, found, err := unstructured.NestedFieldNoCopy(u.Object, "spec", "conversion", "webhookClientConfig", "service")
```

Comment:
nit, can we write is as 
```
service, found, err := unstructured.NestedFieldNoCopy(u.Object, "webhookClientConfig", "service")
```
or
```
service, found, err := unstructured.NestedFieldNoCopy(u.Object, "conversion", "webhookClientConfig", "service")```

Reply:
as the service available in `spec.conversion.webhookClientConfig.service` we will need to fetch spec and then put that as unstructured.spec and can do `conversion.webhookClientConfig.service` 
please do let me know if we can do something else with this.

Reply:
sure, please try this. 
we don't have to specify the entire path from the root.


in this pr itself, in rolebinding transformer we didn't have to use `"spec", "subjects"` 

i believe the path just has to be `distinct enough` but now absolute.
if you think using full path is more readabe, then lets use that.
```
subjects, found, err := unstructured.NestedFieldNoCopy(u.Object, "subjects")
```